### PR TITLE
Add pipeline to test upgrades on azure-operator

### PR DESCRIPTION
**Pull request by fiunchinho**

### CODE REVIEW

File: tekton/pipelines/azure-operator-upgrade.yaml [hunk 1]
```diff
@@ -0,0 +1,124 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: azure-operator
```
Comment (whites11) [inline tekton/pipelines/azure-operator-upgrade.yaml:4]: You should rename it

Comment (fiunchinho) [reply tekton/pipelines/azure-operator-upgrade.yaml]: Fixed.d

**Review by whites11 — APPROVED** [review]
Seems reasonable

**Review by tuommaki — APPROVED** [review]
Can't say much about the change but LGTMIIWFY :slightly_smiling_face:

**Pull request merged**